### ISSUE TITLE
Unable to take two desktop session using mesh central to windows server

### ISSUE THREAD

**Issue by Yogesh1190** [Ylianst/MeshCentral#5235]
**Describe your issue**
I have created windows server 2019 on which I have installed MeshAgent. I am able to see the device in MeshUI for taking remote session.
Now on my server there are two users present for which I am able to take remote session using windows Remote Desktop Connection tool.
1. Yogesh
2. Shivansh

I have taken RDP session using windows Remote Desktop Connection tool for both the user at the same time. Below is the screenshot for your reference,

![image](https://github.com/Ylianst/MeshCentral/assets/19817600/b6ad8089-f474-4099-9242-8e4d6499ada2)

I login to Mesh UI and try to connect to same device using connect option. I am getting below list of using which is expected.

![image](https://github.com/Ylianst/MeshCentral/assets/19817600/8c399afe-ac14-474d-a933-474328493d3a)

Now if I select user Yogesh then I am able to take desktop screen for same user. At the same time if I am taking one more session for user Shivansh then its connecting to Yogesh user only. Always subsequent user wanted to take remote session on the same machine but with different user then its connecting to first connected session only.

Is this bug in MeshCentral ?? Or I need to configure any more properly in config.json file to achieve this scenario. Kindly help as soon as possible.

@Ylianst @krayon007

**Comment by si458** [comment]
how do u open up seperate sessions to each user?

if you HOLD the SHIFT KEY when you select a computer it should open up a seperate window
then in the seperate window you should be able to select connect and connect to a different user?
im assuming!? (i dont use RDP multi user setup so im not too sure im afraid)

**Comment by Yogesh1190** [comment]
I am opening two different tab on the same browser. And then connecting the same server. Below is screenshot for reference,

![image](https://github.com/Ylianst/MeshCentral/assets/19817600/1ed14079-4457-4dfe-a75b-6a86e5b54141)

Tab One connecting to user **Yogesh**
Tab Two connecting to user **Shivansh**

**Expected result** : I should be able to take desktop session for these two user at the same time.

**Actual Result** : If I have already connected to user **Yogesh** for Mesh Desktop Session then though I select user **Shivansh**, it connect me to first connected user only.

**Comment by si458** [comment]
strange request, have u tried connecting to one user in one browser, then connecting to other user in a different browser?
(so user 1 in chrome, user 2 in firefox?)

**Comment by Yogesh1190** [comment]
Yes. I have tried that as well. Also I tried to access form the different machine.

Server having two user

1. UserOne
2. UserTwo

LaptopOne tries for UserOne and able to take remote session for UserOne

LaptopTwo tries for UserTwo and session connecting for UserOne.

**Comment by Yogesh1190** [comment]
@si458 @krayon007 @Ylianst Is this known issue in Mesh or I am missing any configuration to make this work.

**Comment by Mobi974Soft** [comment]
Don' you need 2 CAL for windows server ?

**Comment by jirijanata** [comment]
Hi, have you tried to change DesktopMultiplex Parameter? In this video it's described:

https://www.youtube.com/watch?v=r0IfzuZgf3s

I'm not sure if it also applies for the RDP sessions but I think it's worth to try. I tried it in my setup and it doesn't helped, but you can try it in your setup if it will change anything. 

`"desktopMultiplex": true,`

**Comment by NickBootOne** [comment]
#4695

**Comment by NickBootOne** [comment]
#4695
